### DISCUSSION TITLE
getting trailin comma error

### DISCUSSION THREAD

**Discussion by ferater** [lllyasviel/Fooocus#2883] [Q&A] [answered]
File "/content/Fooocus/modules/async_worker.py", line 50
    from modules.sdxl_styles import get_random_style, random_style_name, apply_style, apply_wildcards, fooocus_expansion, apply_arrays,
                                                                                                                                       ^
SyntaxError: trailing comma not allowed without surrounding parentheses

**Comment by mashb1t** [comment, marked as answer]
make sure that if you did changes to the code to fall back to the official implementation on errors: https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L46

**Reply by mashb1t** [reply]
@ferater it is only now that i recognized you're using my fork https://github.com/mashb1t/Fooocus!
The issue has been fixed in the fork now when merging new features, thanks for reporting.
